packages/dds/tree/src/shared-tree-core/editManagerSummarizer.ts

```diff
@@ -30,12 +30,7 @@ import {
 	ChangeEncoder,
 	ChangeFamilyEditor,
 } from "../core";
-import {
-	Index,
-	SummaryElement,
-	SummaryElementParser,
-	SummaryElementStringifier,
-} from "../shared-tree-core";
+import { Summarizable, SummaryElementParser, SummaryElementStringifier } from ".";
```

Comment:
generally import from "." is bad as it causes cyclers. use a more specific import (can we make our linter ban that?)

Reply:
Good catch, thanks. I agree it should be banned.